src/main/java/net/cyberkitsune/prefixchat/ChatListener.java

```diff
@@ -18,8 +17,11 @@
 
 	public ChatListener() {
 		this.bufs = new HashMap<>();
+		dashReminded = new ArrayList<String>();
 	}
 
+	private List<String> dashReminded;
```

Comment:
Users are added to list but never removed. Please find a place to remove users from list.

Reply:
added playerquitevent to remove them from dashreminded list